src/templates/muc_nickname_form.js

```diff
@@ -15,8 +15,13 @@ export default (o) => {
             <form class="converse-form chatroom-form converse-centered-form">
                 <fieldset class="form-group">
                     <label>${i18n_heading}</label>
-                    <input type="text" required="required" name="nick" value="${o.nick || ''}"
-                        class="form-control" placeholder="${i18n_nickname}"/>
+                    <p class="validation-message">${o.validation_message}</p>
```

Comment:
The validation message gets saved as `nickname_validation_message`, not `validation_message`.